gst/tensor_aggregator/tensor_aggregator.c

```diff
@@ -757,7 +757,7 @@ gst_tensor_aggregator_concat (GstTensorAggregator * self, GstBuffer * outbuf,
 
   do {
     for (f = 0; f < self->frames_out; f++) {
-      memcpy (dest_info.data + dest_idx,
+      orc_memcpy (dest_info.data + dest_idx,
           src_info.data + src_idx + (frame_size * f), block_size);
```

Comment:
I just wonder.. How good it is?

Reply:
https://gstreamer.freedesktop.org/documentation/design/orc-integration.html

Reply:
We need a static-inline or macro function that calls memcpy if the size if small. (e.g., 100)